nwb_conversion_tools/datainterfaces/blackrockdatainterface.py

```diff
@@ -46,9 +47,9 @@ def get_metadata(self):
         comment = nsx_file.basic_header['Comment']
 
         metadata['NWBFile'] = dict(
-            session_start_time=session_start_time_tzaware,
+            session_start_time=session_start_time_tzaware.strftime('%Y-%m-%dT%H:%M:%S'),
             session_description=comment,
-            identifier=''.join(random.choices(string.ascii_uppercase + string.digits, k=12))
+            identifier=str(uuid.uuid4())
```

Comment:
Again, [these lines called by the primary NWBConverter](https://github.com/catalystneuro/nwb-conversion-tools/blob/54608fd49a7eff977cfb9d69d5cd73e90e78e927/nwb_conversion_tools/conversion_tools.py#L44) mean that any other use of the identifier field (especially within interfaces) is redundant.

And in general principle, is really something that we should always leave to the main converter anyway.